### PR TITLE
B-18648, B-18855, and B-18851

### PR DESCRIPTION
**Pull request by WeatherfordAaron**
There was some overlap between changes on the PRs associated with B-18855 and B-18648.
I have included changes on PRs associated with B-18851 because that story falls under the same feature as B-18648.

## PR #12352 [link](https://github.com/transcom/mymove/pull/12352)

```
12449 12428 12372 12325 12461 12443 12352
```

### CODE REVIEW

File: src/components/Office/ServiceItemCalculations/ServiceItemCalculations.jsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { makeCalculations } from './helpers';
 import styles from './ServiceItemCalculations.module.scss';
 
 import { PaymentServiceItemParam, MTOServiceItemShape } from 'types/order';
-import { allowedServiceItemCalculations } from 'constants/serviceItems';
+import { allowedServiceItemCalculations, SERVICE_ITEM_CALCULATION_LABELS } from 'constants/serviceItems';
```
Comment (taeJungCaci) [inline src/components/Office/ServiceItemCalculations/ServiceItemCalculations.jsx:10]: Missing `SERVICE_ITEM_CODES` and other lines of code in this file. Checked against integrationTesting branch.

Comment (WeatherfordAaron) [reply src/components/Office/ServiceItemCalculations/ServiceItemCalculations.jsx:10]: ```SERVICE_ITEM_CODES``` is not needed on this one. I think it's possible some other code that is in INT and not a part of this ticket could be using it.. basically if I add that in, the linter says that it's declared and not used.

Comment (taeJungCaci) [reply src/components/Office/ServiceItemCalculations/ServiceItemCalculations.jsx:10]: Last note before someone else reviews, this code was used in the file and you have added it:

image

File: src/utils/formatters.js [hunk 2]
```diff
@@ -523,6 +523,15 @@ export function formatTimeUnitDays(days) {
...
+  let result = '';
+  if (withCommas === true) {
+    result = `${distance.toLocaleString()}`;
+  } else {
+    result = `${distance}`;
+  }
+  if (withUnit === true) {
+    result += ' miles';
+  }
+  return result;
```
Comment (brooklyn-welsh) [inline src/utils/formatters.js:536]: It looks like this code block is present here but didn't make it into IntegrationTesting, just checking if this was intentional of if it needs a quick INT PR?

Comment (WeatherfordAaron) [reply src/utils/formatters.js:536]: lemme remove.. it was supposed to be removed

Comment (WeatherfordAaron) [reply src/utils/formatters.js]: ok should be back to normal now.

**Comment by JamesHawks224** [comment]
updated verify rpt:
[B18648_B18855_B18851-12550.verify_rpt.txt](https://github.com/transcom/mymove/files/15067809/B18648_B18855_B18851-12550.verify_rpt.txt)

alot of diffs

**Comment by WeatherfordAaron** [comment]
> updated verify rpt: [B18648_B18855_B18851-12550.verify_rpt.txt](https://github.com/transcom/mymove/files/15067809/B18648_B18855_B18851-12550.verify_rpt.txt)
> 
> alot of diffs

Due to the number of kickbacks this one has had, this report may lose some accuracy. The way I was able to check it was to mainly look at the last PR's (#12461) diffs (which were fixed with Tae's and Brooklyn's suggestions.

**Review by brooklyn-welsh — APPROVED** [review]
Looks like the requested changes were made, LGTM ☑️

**Review by JamesHawks224 — APPROVED** [review]
LGTM

**Pull request merged**